tests/vmi_lifecycle_test.go

```diff
@@ -979,11 +975,12 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				config.CPUModel = defaultCPUModel
 				tests.UpdateKubeVirtConfigValueAndWait(*config)
 
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-				vmi.Spec.Domain.CPU = &v1.CPU{
-					Model: vmiCPUModel,
-				}
-				_, err = virtClient.VirtualMachineInstance(vmi.Namespace).Create(vmi)
+				vmi := libvmi.NewCirros(
+					libvmi.WithCPU(&v1.CPU{
+						Model: vmiCPUModel,
+					}),
```

Comment:
nit: I think this can be a oneliner, like used later on.

Reply:
changed, thank you.